package.json

```diff
@@ -5,8 +5,19 @@
   "repository": "https://github.com/riskycase/my-express",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+	"pretest": "eslint --ignore-path .gitignore ."
```

Comment:
How you test and fixed the issues in your PR?

Reply:
Test returned zero errors as eslint is configured to use es6 rules in .eslintrc